src/integrationTest/resources/application-test.yaml

```diff
@@ -1,9 +1 @@
-spring:
-  datasource:
-    driver-class-name: org.h2.Driver
-    url: jdbc:h2:mem:testdb
-    username: my-db
-    password: my-db
-    embedded-directory: build
-    jpa:
-      spring.jpa.database-platform: org.hibernate.dialect.H2Dialect
+
```

Comment:
probs best just deleting the file if you can?

Reply:
Looks like one of the integration tests requires that jsonpath bit from the bottom of this file. Any suggestions?